symengine/eval_mpc.cpp

```diff
@@ -265,6 +265,23 @@ class EvalMPCVisitor : public BaseVisitor<EvalMPCVisitor> {
         }
     }
 
+    void bvisit(const Gamma &x) {
```

Comment:
mpc doesn't has `gamma` implementation. So I guess, this should throw error for now.